include/spdlog/logger.h

```diff
@@ -368,11 +369,7 @@ class SPDLOG_API logger
         SPDLOG_TRY
         {
             memory_buf_t buf;
-#ifdef SPDLOG_USE_STD_FORMAT
             fmt_lib::vformat_to(std::back_inserter(buf), fmt, fmt_lib::make_format_args(std::forward<Args>(args)...));
```

Comment:
I think fmt lib’s back_inserter optimization is not yet released (https://github.com/gabime/spdlog/issues/2485#issuecomment-1245783205)

Reply:
Will add back. I initially removed because the wide character version had the same code when using fmtlib and when not using it.